Riot/Views/Calls/IncomingCallView.m

```diff
@@ -0,0 +1,323 @@
+/*
+ Copyright 2017 Vector Creations Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "IncomingCallView.h"
+
+#import <MatrixKit/MXKImageView.h>
+#import <MatrixSDK/MXMediaManager.h>
+
+#import "CircleButton.h"
+
+static const CGFloat kAvatarSize = 100.0;
+static const CGFloat kButtonSize = 80.0;
+
+@interface IncomingCallView ()
+
+@property (nonatomic) MXKImageView *callerImageView;
+@property (nonatomic) UILabel *callerNameLabel;
+@property (nonatomic) UILabel *callInfoLabel;
+
+@property (nonatomic) CircleButton *answerButton;
+@property (nonatomic) UILabel *answerTitleLabel;
+
+@property (nonatomic) CircleButton *rejectButton;
+@property (nonatomic) UILabel *rejectTitleLabel;
+
+@end
+
+@implementation IncomingCallView
+
++ (CGSize)callerAvatarSize
+{
+    return CGSizeMake(kAvatarSize, kAvatarSize);
+}
+
+- (instancetype)initWithCallerAvatarURL:(NSString *)callerAvatarURL placeholderImage:(UIImage *)placeholderImage callerName:(NSString *)callerName callInfo:(NSString *)callInfo
+{
+    self = [super initWithFrame:CGRectZero];
+    if (self)
+    {
+        self.backgroundColor = [UIColor whiteColor];
```

Comment:
We have dark theme now.
Instead of using hard-coded colours, you should use variables defined at https://github.com/vector-im/riot-ios/blob/master/Riot/Utils/RiotDesignValues.h#L35